### PR TITLE
refactor: doobie to skunk migration

### LINKED ISSUE #219: Replace Doobie with Skunk
As a Renku owner, I'd like Graph Services to use Skunk instead of Doobie for accessing Postgres DB.

Acceptance criteria:
* Replace dependency to Doobie with Skunk and perform all the necessary changes so everything works as before;
* new issue - Use Skunk's notify functionality in the Events Processing flow;
* new issue - Use Skunk's notify functionality for the Metrics;
* new issue - Create an issue for creating a WebSocket version of the endpoint providing info about the processing status;

### PR DESCRIPTION
**Pull request by joke1196**
closes #219

### CODE REVIEW

File: event-log/src/main/scala/io/renku/eventlog/Microservice.scala [hunk 1]
```diff
@@ -57,19 +58,19 @@ object Microservice extends IOMicroservice {
   protected implicit override def timer: Timer[IO] = IO.timer(executionContext)
 
   override def run(args: List[String]): IO[ExitCode] = for {
-    transactorResource <- new EventLogDbConfigProvider[IO](args) map DbTransactorResource[IO, EventLogDB]
+    transactorResource <- new EventLogDbConfigProvider[IO]() map SessionPoolResource[IO, EventLogDB]
```
Comment (jachro) [inline event-log/src/main/scala/io/renku/eventlog/Microservice.scala:61]: Maybe `transactorResource` should be renamed to `sessionPoolResource`? Probably the same could be done on token-repository's Microservice

File: event-log/src/main/scala/io/renku/eventlog/processingstatus/ProcessingStatusFinder.scala [hunk 2]
```diff
@@ -19,52 +19,59 @@
...
-                  |    ORDER BY batch_date DESC
-                  |    LIMIT 1
-                  |  ) max_batch_date ON evt.batch_date = max_batch_date.batch_date
-                  |WHERE evt.project_id = $projectId
-                  |""".stripMargin.query[EventStatus].to[List],
+  private def latestBatchStatues(projectId: Id) = SqlQuery[Interpretation, List[EventStatus]](
+    Kleisli { session =>
+      val query: Query[Id ~ Id, EventStatus] =
+        sql"""SELECT evt.status
+                                                     FROM event evt
```
Comment (jachro) [inline event-log/src/main/scala/io/renku/eventlog/processingstatus/ProcessingStatusFinder.scala:63]: Could we format that a bit better

File: graph-commons/src/test/scala/ch/datascience/db/DbConfigGenerator.scala [hunk 3]
```diff
@@ -29,18 +29,19 @@ import scala.language.postfixOps
...
-    user           <- nonEmptyStrings() map (RefType.applyRef[User](_).getOrError)
-    pass           <- nonEmptyStrings()
-    connectionPool <- positiveInts() map (_.value) map (RefType.applyRef[ConnectionPool](_).getOrError)
-    maxLifetime    <- durations(max = 60 minutes)
-  } yield DBConfig(driver, url, user, pass, connectionPool, maxLifetime)
-
-  implicit class RefinedOps[V](maybeValue: Either[String, V]) {
-    lazy val getOrError: V = maybeValue.fold(s => throw new IllegalArgumentException(s), identity)
-  }
+//  val dbConfigs: Gen[DBConfig[TestDB]] = for {
```
Comment (jachro) [inline graph-commons/src/test/scala/ch/datascience/db/DbConfigGenerator.scala:32]: Should that go?

Comment (joke1196) [reply graph-commons/src/test/scala/ch/datascience/db/DbConfigGenerator.scala:32]: Yes good catch everything is in the newConfig now as we cannot fake a connection

**Review by jachro — APPROVED** [review]

**Pull request merged**